### ISSUE TITLE
New note picker

### ISSUE THREAD

**Issue by hansaplasst** [SimpleMobileTools/Simple-Notes#674]
Hi, 

I'm not fond of the new note picker view where all the notes are partly visible. It shows too much info which makes it cluttered imho. The old view only contained a list. This made it much faster to scroll through notes.

Any chance of adding a picker-list-view-option in the settings?

**Comment by xcelente** [comment]
Yes, this is the exact reason I came here, to create this very issue. I have many notes and it's just so clumsy to open notes this way.

**Comment by SuzaBr** [comment]
First of all, Thank you @tibbi for making this wonderful app. I have used it for years now and it has always worked very well! 

Still, with the new grid picker the app is unusable for me. It takes ca. 5 seconds to open on my phone (quite old Samsung). Before the change it was < 1 second. There are 39 notes in the app.

I seriously tried to to get used to the new grid layout. However since the picker was introduced, I find myself using simple notes less and less, as it is not simple to use for me anymore. So my verdict after 2 months of testing is that the old picker was way superior (at least on a phone that's a bit older).

Some questions:
- Would it hurt to just revert the change?
  -  Perfection is also, when nothing can be removed anymore, [someone](https://www.goodreads.com/quotes/19905-perfection-is-achieved-not-when-there-is-nothing-more-to) said.
- I wonder why the change was introduced in the first place?
  - I can only find one feature request by a deleted account: https://github.com/SimpleMobileTools/Simple-Notes/issues/641
  - It has no reasoning, just "Current UI is terrible" and received no likes.

**Comment by xcelente** [comment]
As you can see, my last comment about this was 3 months ago. In this time I have intentionally given myself ample time to get used to this new note picker, plus the revised smaller preview. I have concluded that this note picker is a step backwards. Sorry, I don't mean to sound negative, only that I have really tried to challenge my own opinion here.

I currently have 37 notes / checklists and having to scroll throw them just to switch is a pain. Plus, the note picker always starts at the top and not from where I am in the list. In most cases I know what the notes are about, I don't need a preview.

What I want is the whole screen to fill with a simple single column list of note titles. Currently, the note picker floats, not occupying the entire phone screen. Give me functionality over looks please. If I really need to find something, then give me a global search tool.

Hoping someone re-considers this. Thanks.

**Comment by inson1** [comment]
@xcelente SMT apps were sold. Fork is here https://github.com/FossifyOrg First release in 1-2 weeks
